### DISCUSSION TITLE
Option to close everything from current session before loading another one

### DISCUSSION THREAD

**Discussion by Martin1887** [olimorris/persisted.nvim#90] [Ideas]
Firstly, thanks for this great extension!

I have not missed anything except the ability to destroy a session (saving it first) manually and/or automatically before loading another one. The use case is being working in a workspace in an opened session and wanting to change to a different one. One option for this is to close Neovim and open it in the folder of the desired session but having an option to do it without leaving Neovim would be great!

Thanks.

**Comment by olimorris** [comment]
Thank you and glad you're enjoying it.

Have you checked out the [events](https://github.com/olimorris/persisted.nvim#events--callbacks)? You could most certainly do this with the Telescope events. Let me know if you have any questions on those. I probably should add some better examples to the readme.

**Reply by Martin1887** [reply]
I guess the event `PersistedTelescopeLoadPre` can be used, but anyways I think it is a common enough feature to be built-in as an option/command. Thanks for your quick response.

**Reply by olimorris** [reply]
I'm keen to keep the code opinion free but will put the example in the Readme 👍🏼

**Reply by olimorris** [reply]
I've updated the Readme now. Let me know if that's what you had in mind.

**Reply by Martin1887** [reply]
Perfect, thanks!
